src/Microsoft.Bcl.HashCode/tests/HashCodeSeedInitializerTests.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.RemoteExecutor;
+using System;
+using System.Reflection;
+using Xunit;
+
+namespace Microsoft.Bcl.HashCode.Tests
+{
+    public class HashCodeSeedInitializerTests
+    {
+        [Fact]
+        public void EnsureSeedReturnsDifferentValuesTest()
+        {
+            var executor1 = RemoteExecutor.Invoke(GetHashCodeSeed, new RemoteInvokeOptions() { CheckExitCode = false });
+
+            int FirstSeed = executor1.ExitCode;
+            executor1.Dispose();
+
+            var executor2 = RemoteExecutor.Invoke(GetHashCodeSeed, new RemoteInvokeOptions() { CheckExitCode = false });
+
+            int SecondSeed = executor2.ExitCode;
+            executor2.Dispose();
+
+            Assert.NotEqual(FirstSeed, SecondSeed);
+
+            int GetHashCodeSeed()
+            {
+                var seed1Field = typeof(System.HashCode).GetField("s_seed", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
Rather than use reflection to read an internal field can you just call the API with the same input and make sure you get different results?  

You could also do the same in order to test the switch you’ve added.

Reply:
Here's what I was thinking:
```c#

            HashCode hc = new HashCode();
            hc.Add("Hello");
            hc.Add("World");
            return hc.ToHashCode();
```

Currently on desktop this will always return `1902986306`, but your change should fix that.

You can also have a test which only applies to .NETFramework, sets the app-context switch, and ensures that the subsequent calls return the same result.

Reply:
Here's an example that shows how to test your appcontext switch: https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Runtime/tests/System/UseResourceKeysTest.cs#L15

Reply:
oh just saw your comments I had already did something very similar to what you pointed out.

Reply:
I can't use RuntimeConfigurationOptions because that doesn't exist in netfx, but I'm able to just call AppContext APIs to manually set the switch before calling hashcode so I'll do that.